lang/fr.json

```diff
@@ -284,7 +284,7 @@
     "description": "Ideas tab button text. Tab shows programs to give students ideas."
   },
   "ideas-tab-description": {
-    "defaultMessage": "Espace réservé pour les exemples de code complet",
+    "defaultMessage": "Espace réservé pour des exemples de code complètement fonctionnels",
```

Comment:
I think we expect a different string here. What's happened?

Reply:
In English this string is now: "Try out these projects, modify them and get inspired".
Perhaps this isn't reflected in Crowdin.

Reply:
I don't understand how, but Crowdin still has the old English here. Generally the file looks up-to-date though (e.g. the "add" and "micro:bit ready" changes.

Reply:
That string is the only discrepancy between the files. I'll upload the source to Crowdin and see if it updates. Perhaps I just failed to upload it after it changed.

Reply:
We knowingly added this string (and all of Ideas) after translation. However, it's worth getting this done somehow now it's the only one.

Reply:
We'll sort a machine translation and get some internal review. Resolving.